paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTabLayoutUI.kt

```diff
@@ -388,9 +372,7 @@ private fun SavedPaymentMethodTab(
         SavedPaymentMethodTab(
             viewWidth = width,
             editState = when {
-                isEnabled && isEditing && (isModifiable || FeatureFlags.useNewUpdateCardScreen.isEnabled) ->
-                    PaymentOptionEditState.Modifiable
-                isEnabled && isEditing -> PaymentOptionEditState.Removable
+                isEnabled && isEditing -> PaymentOptionEditState.Modifiable
```

Comment:
Looks like we don't even need an extra boolean, this looks like it's always just isEditing.

Reply:
I _think_ that `isEnabled` is used to ensure that we don't show the edit icon icon for Link or the "Add card" tab, but lmk if I'm missing something